scripts/mobile-cv-suite-config.cmake

```diff
@@ -48,11 +58,11 @@ if (_MCS_USE_SLAM)
   list(APPEND _MCS_INTERFACE_LIBS
     # --- static
     ${_MCS_LIBS}/libamd.a
-    ${_MCS_LIBS}/libbtf.a
+    ${_MCS_LIBS}/libbtf${CMAKE_SHARED_LIBRARY_SUFFIX}
```

Comment:
Is this okey? It seemed to work fine with both iOS and Mac builds

Reply:
libbtf and cxsparse are LGPL licensed so they must be used as shared libraries, so that change is good. Perhaps move them from under the "--- static" heading to "--- shared" and mention the LGPL thing.